backend/api/EventHandlers/MissionEventHandler.cs

```diff
@@ -109,24 +107,16 @@ private async void OnMissionRunCreated(object? sender, MissionRunCreatedEventArg
 
         private async void OnRobotAvailable(object? sender, RobotAvailableEventArgs e)
         {
-            _logger.LogInformation(
-                "Triggered RobotAvailable event for robot ID: {RobotId}",
-                e.RobotId
-            );
-            var robot = await RobotService.ReadById(e.RobotId, readOnly: true);
-            if (robot == null)
+            if (e.Robot == null)
```

Comment:
Can the robot be required to be a part of the event args so this can be known to never be null?

Reply:
I will remove the null check. Seems like it will never be null